src/sentry/utils/types.py

```diff
@@ -116,7 +116,7 @@ class StringType(Type):
 
 
 class DictType(Type):
-    """Coerce a dict out of a json/yaml string"""
+    """Coerce a dict out of a yaml string"""
```

Comment:
I'd argue to revert the changes here to the docstrings, since json strings are still valid. And it's not overly obvious to someone that yaml is a superset of json.